guardian/support-frontend
# Remove unused dev dependencies

## What are you doing in this PR?

Remove unused dev dependencies (branching off of #6663) according to knip and adding the `dependencies` option to the script, which would report newly unused dependencies from the github action (#6654 )



[**Trello Card**](https://trello.com/c/cd3vdNVV/1298-knip-report-on-unused-dependencies-%E2%9C%82%EF%B8%8F)




## How to test

- All checks pass
- Deploy to code and check that pages look correct
- Run playwright tests
